app/src/main/cpp/ExternalBlitter.cpp

```diff
@@ -160,6 +160,7 @@ ExternalBlitter::CancelFrame(const int32_t aSurfaceHandle) {
     surface = iter->second;
   } else {
     surface = GeckoSurfaceTexture::Create(aSurfaceHandle);
+    m.surfaceMap[aSurfaceHandle] = surface;
```

Comment:
What condition causes us to have a `aSurfaceHandle` value with out it being in the `m.surfaceMap`?

Reply:
A new texture that hasn't been used before or StopPresenting() call which cleared the surfaces